.circleci/config.yml

```diff
@@ -77,25 +66,19 @@ defaults: &defaults
         # work ok, but given that these CircleCI runners aren't the beefiest
         # machine can cause some flakiness. So pass -j1 to Tasty (NOT Stack) to
         # tell it to go slow and steady.
-        command: stack --stack-yaml=${STACK_FILE} test haskell-language-server:func-test --dump-logs --test-arguments="-j1 --rerun-update" || stack --stack-yaml=${STACK_FILE} test haskell-language-server:func-test --dump-logs --test-arguments="-j1 --rerun" || LSP_TEST_LOG_COLOR=0 LSP_TEST_LOG_MESSAGES=true LSP_TEST_LOG_STDERR=true stack --stack-yaml=${STACK_FILE} test haskell-language-server:func-test --dump-logs --test-arguments="-j1 --rerun"
+        command: stack --stack-yaml=${STACK_FILE} test --no-run-tests haskell-language-server:func-test --dump-logs --test-arguments="-j1 --rerun-update" || stack --stack-yaml=${STACK_FILE} test haskell-language-server:func-test --dump-logs --test-arguments="-j1 --rerun" || LSP_TEST_LOG_COLOR=0 LSP_TEST_LOG_MESSAGES=true LSP_TEST_LOG_STDERR=true stack --stack-yaml=${STACK_FILE} test haskell-language-server:func-test --dump-logs --test-arguments="-j1 --rerun"
         no_output_timeout: 120m
```

Comment:
--no-run-tests is only added to the first of 3 invocations?

Reply:
good point! I have remove the subsequent invocations now

Reply:
(You could delete the comment about why we are using -j1 as well, if you wanted.)